File: QTask/Models/Task.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace QTask.Models
{
    public class Task
    {
        public long TaskId { get; set; }
        public string Name { get; set; }
        public bool IsCompleted { get; set; }

        public long ProjectId { get; set; }
        public string ExecutorId { get; set; }

        public virtual Project Project { get; set; }
        public virtual ApplicationUser Executor { get; set; }
    }
}

File: QTask/src/app/models/task.ts
import { User } from './user'; 

export class Task {
    name: string;
    description: string;
    state: string;
    authorId: number;
    executorId: number;

    constructor(name: string, description: string, state: string, authorId: number, executorId: number) {
        this.name = name;
        this.description = description;
        this.state = state;
        this.authorId = authorId;
        this.executorId = executorId;
    };
}